apps/pie-storybook/stories/testing/icons.test.stories.ts

```diff
@@ -0,0 +1,122 @@
+import { html } from 'lit/static-html.js';
+import '@justeattakeaway/pie-icons-webc';
+import {
+    regularIconSizes, largeIconSizeDefault, regularIconSizeDefault, largeIconSizeModule, type LargeIconSize, type RegularIconSize,
+} from '@justeattakeaway/pie-icons-configs';
+import { createStory, createVariantStory, type TemplateFunction } from '../../utilities';
+
+const iconsStoryMeta = {
+    title: 'Icons',
+    parameters: {
+        layout: 'fullscreen',
+    },
+};
+
+export default iconsStoryMeta;
+
+type RegularIconProps = {
+    size: RegularIconSize
+}
+
+const defaultRegularIconProps: RegularIconProps = {
+    size: regularIconSizeDefault,
+};
+
+type LargeIconProps = {
+    size: LargeIconSize
+}
+
+const defaultLargeIconProps: LargeIconProps = {
+    size: largeIconSizeDefault,
+};
+
+const AlcoholFilledRegularIconTemplate: TemplateFunction<RegularIconProps> = ({ size }) => html`
+    <div class="c-iconGrid">
+        <icon-alcohol-filled size=${size}></icon-alcohol-filled>
+    </div>
+`;
+
+const AlcoholFilledRegularIconWithOverrideTemplate: TemplateFunction<RegularIconProps> = ({ size }) => html`
+    <div style="--icon-size-override: 200px;">
+        <p>size: ${size}</p>
+        <p>--icon-size-override: 200px</p>
+        <icon-alcohol-filled size=${size}></icon-alcohol-filled>
+    </div>
+`;
+
+const AlcoholFilledLargeIconTemplate: TemplateFunction<LargeIconProps> = ({ size }) => html`
+    <p>For large icons, size must be a number greater than or equal to \`${largeIconSizeDefault}\`, and a multiple of \`${largeIconSizeModule}\`.</p>
```

Comment:
Why would this `<p>` tag description be needed in a story for automated testing? This is the kind of thing I think can be removed. I don't think we should be simply cloning the existing stories as they are for documentation. These can likely be stripped down.